### ISSUE TITLE
How to disable/fix Suspicious state from syntax checker python-mypy message in emacs

### ISSUE THREAD

**Issue by avatar-lavventura** [python/mypy#9784]
On `emacs`, I keep having following warning message, even I have the recent version of `python-mypy` not sure why this warning message is generated; is there any way to fix this, or just disable it?

```
Suspicious state from syntax checker python-mypy: Flycheck checker python-mypy returned 2, but its output contained no errors:
utils.py: error: Source file found twice under different module names: 'Program.utils' and 'utils'
Found 1 error in 1 file (checked 1 source file)

Try installing a more recent version of python-mypy, and please open a bug report if the issue persists in the latest release.
Thanks!
```

-----

pip-setup:

```
$ pip freeze | grep mypy
flake8-mypy==17.8.0
mypy==0.790
mypy-extensions==0.4.3
```

output of `elpy-config` within the `emacs`:

```
Elpy Configuration

Emacs.............: 26.3
Elpy..............: 1.34.0
Virtualenv........: venv (/home/alper/venv)
Interactive Python: python3 3.7.5 (/home/alper/venv/bin/python3)
RPC virtualenv....: rpc-venv (/home/alper/.emacs.d/elpy/rpc-venv)
 Python...........: python 3.7.5 (/home/alper/.emacs.d/elpy/rpc-venv/bin/python)
 Jedi.............: 0.17.2
 Rope.............: 0.16.0
 Autopep8.........: 1.5.4
 Yapf.............: 0.30.0
 Black............: 20.8b1
Syntax checker....: flake8 (/home/alper/venv/bin/flake8)
```

**Comment by hauntsaninja** [comment]
Unrelated to emacs, see https://github.com/python/mypy/issues/8944#issuecomment-640656741 and other comments on that issue. Ask on Gitter if you need more help

**Comment by avatar-lavventura** [comment]
@hackaugusto So do I have to remove the `__init__.py` file on the base directory?

**Comment by hackaugusto** [comment]
@avatar-lavventura you probably at'ed the wrong handle.

**Comment by avatar-lavventura** [comment]
@hackaugusto `at'ed` ?
